.github/workflows/ci.yaml

```diff
@@ -37,6 +37,15 @@ jobs:
       - name: Check static binary size
         if: matrix.system == 'x86_64-linux'
         run: nix run .#check-closure-size
+      - name: nix build
+        if: matrix.system != 'x86_64-darwin'
+        run: nix build
+      - name: Upload om static binary
+        if: matrix.system != 'x86_64-darwin'
+        uses: actions/upload-artifact@v4
+        with:
+          name: om-${{ matrix.system }}
+          path: ./result/bin/om
```

Comment:
Let's use `nix build --no-link --print-out-path` and then pass that path between github steps (see [here](https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable)).

[Explicit is better than implicit.](https://peps.python.org/pep-0020/)

Reply:
Done